tests/tests_suite_test.go

```diff
@@ -57,6 +36,32 @@ type App struct {
 	URL  string
 }
 
+type TestData struct {
+	Username      string
+	Password      string
+	Email         string
+	KeyName       string
+	KeyPath       string
+	Profile       string
+	ProfilePath   string
+	ControllerURL string
+}
+
+var adminTestData TestData
+var testRoot, testHome, keyPath, gitSSH string
```

Comment:
@vdice I think it's here: https://github.com/jchauncey/workflow-e2e/blob/random-user/tests/tests_suite_test.go#L115